src/msg/async/rdma/RDMAServerSocketImpl.cc

```diff
@@ -28,7 +28,7 @@ int RDMAServerSocketImpl::listen(entity_addr_t &sa, const SocketOptions &opt)
   if (server_setup_socket < 0) {

```

Comment:
modify as the following：set “rc = server_setup_socket;” ？
if (server_setup_socket < 0) {
    rc = server_setup_socket;
    lderr(cct) << __func__ << " failed to create server socket: "
               << cpp_strerror(rc) << dendl;
    return rc;
  }